### ISSUE TITLE
[BUG] Accessibility Insights Tool-Entire app function does not work on Win11 24H2.

### ISSUE THREAD

**Issue by JoyZhang6** [microsoft/accessibility-insights-windows#1837]
We found that Accessibility Insights Tool-Entire app function is not work on Win11 24H2.
**Steps to reproduce: **
1.	Download and install the latest version of the Microsoft Accessibility Insights for Windows tool on Win11 24H2 OS: [Accessibility Insights Downloads] (https://accessibilityinsights.io/downloads/)
2.	Launch the tool
3.	From the left pane of the Accessibility Insights tool, select the Settings option (the Gear icon), under the Other Options, change the “Always on Top” settings to OFF and save the change.
4.	On the left pane of the Accessibility Insights tool, select the Inspect Live option.
5.	Change the What to Select option to "Entire app".
6.	Select the Application under Test.  –> issue. There are no highlighted boxes around the Application.

https://github.com/microsoft/accessibility-insights-windows/assets/173444047/967dbb32-ed9a-4ec6-a547-2aeeabca6348

**Comment by JoyZhang6** [comment]
https://github.com/microsoft/accessibility-insights-windows/assets/173444047/2647b09e-4b1c-4a8f-91c8-31cc172a3029

**Comment by JoyZhang6** [comment]
Hi @codeofdusk This issue still exists on the latest Accessibility Insights Tool Version: 1.1.2703.1
 Is there any additional information needed to help with this issue?

**Comment by Michellehdr** [comment]
I saw the same issue on my end.

**Comment by codeofdusk** [comment]
First, our sincerest apologies for the delay in getting this sorted.

I'm pleased to report that the latest production release ([v1.1.2829.01](https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.2829.01)) has a fix for this issue!
